tqftpserv.c

```diff
@@ -138,7 +138,7 @@ static int tftp_send_oack(int sock, size_t *blocksize, size_t *tsize,
 		*p++ = '\0';
 	}
 
-	if (tsize && *tsize != -1) {
+	if (tsize && *tsize != -1U) {
```

Comment:
This is ridiculous IMO

Reply:
hey, I didn't come up with assigning -1 as an error value to a variable of type size_t /shrug